spec/features/job_seekers_can_filter_vacancies_spec.rb

```diff
@@ -56,6 +56,50 @@
     end
   end
 
+  scenario 'Filterable by subject', elasticsearch: true do
+    headmaster = create(:vacancy, :published, job_title: 'Headmaster')
```

Comment:
These can probably be set in a `context` block using `let`s, just to DRY up the tests a bit

Reply:
Done!